src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/DisposeAnalysis/DisposeAnalysis.DisposeDataFlowOperationVisitor.cs

```diff
@@ -97,6 +97,12 @@ protected override DisposeAbstractValue HandleInstanceCreation(IOperation creati
                     return defaultValue;
                 }
 
+                // Handle user option for additional excluded types
+                if (DataFlowAnalysisContext.ExcludedSymbols.Contains(instanceType))
```

Comment:
I am only handling fully qualified types and type names, let me know if you think that's worth also handling namespaces.

Reply:
I think this is fine.